runtimes/spiritnet/src/weights/pallet_balances.rs

```diff
@@ -19,7 +17,9 @@
 //! Autogenerated weights for pallet_collective
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2023-03-10, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2023-05-31, STEPS: `50`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
```

Comment:
Why is this not changed also for the peregrine runtime?

Reply:
I do not know what happened. I recalculated the weights here: [4d040ad](https://github.com/KILTprotocol/kilt-node/pull/541/commits/4d040ad59ea32b76034ee7bb32cc4874789ed243)